### DISCUSSION TITLE
Add option for "bedrock menus"

### DISCUSSION THREAD

**Discussion by Elgorond** [BG-Software-LLC/SuperiorSkyblock2#1557] [Pending Feature Requests]
For java servers that have custom menus (like ours here [https://imgur.com/a/zdBniia](https://imgur.com/a/zdBniia)), there are issues when trying to enable cross-play with Bedrock users. Basically Bedrock doesn't support custom resource pack images in chest menus. Looking at the same menu from java vs bedrock unfortunately looks like this: [https://imgur.com/a/aF5lVst](https://imgur.com/a/aF5lVst)

So I'm proposing that the plugin can optionally display different menus depending on if the player is on Java edition or not. It could detect the version simply by the player name formatting (since Geyser has Bedrock players with a configurable character before their name, like ".notch" or "*notch" for example). So a java player would receive option A Skyblock menus, whereas a bedrock player could receive option B Skyblock menus.

Bedrock is becoming more popular and more servers are supporting cross-play. So I think this is a needed feature for servers that utilize custom textures. Separate bedrock menus can ensure it does not look broken for Bedrock players.

Maximum compatibility for custom textures would involve support for "JSON UI." Personally I know little about the subject, but this is how custom menus are done in bedrock [https://wiki.bedrock.dev/json-ui/json-ui-intro.html#top](https://wiki.bedrock.dev/json-ui/json-ui-intro.html#top). I'm not sure if SuperiorSkyblock ever intends on supporting crossplay maximally, but adding the bedrock menu option would be great for the time being!

**Comment by SinonCute** [comment]
Use api to check instead prefix
